Add breaking change for ldap automated rotation TTLs when a manual rotation is performed (#2923)
hashicorp/web-unified-docs: content/vault/v2.x/content/docs/updates/important-changes.mdx

## Breaking changes

## LDAP static roles do not reset rotation TTL after manual rotation ((#ldap-manual-rotation-ttl-reset))

| Change       | Affected version | Vault edition
| ------------ | ---------------- | -------------
| Breaking     | 2.0.0+           | Enterprise

[LDAP secrets engine static roles](/vault/docs/secrets/ldap) that have automated
credential rotation configured now no longer reset their

> rotation TTL after a manual rotation is performed. The original rotation period

Reviewer: ```suggestion
rotation TTL when you perform a manual rotation. Vault preserves the original rotation period
```
Style correction: write in active voice

> cadence is preserved and will continue to be rotated automatically at the

Reviewer: ```suggestion
cadence and continues to rotate the static role automatically at the
```
Style correction: write in active voice
